templates/includes/sidebar.html

```diff
@@ -0,0 +1,14 @@
+<ul class="sidebar-nav">
+  <li>
+    <a href="{% url 'taxi:index' %}">Home page</a>
+  </li>
+  <li>
+    <a href="#">Manufacturers</a>
+  </li>
+  <li>
+    <a href="#">Cars</a>
+  </li>
+  <li>
+    <a href="#">Drivers</a>
```

Comment:
The links for 'Manufacturers', 'Cars', and 'Drivers' are currently placeholders (`#`). If these sections are intended to have their own pages, consider updating these links with the appropriate URLs using Django's `{% url %}` template tag.